includes/class-comment.php

```diff
@@ -523,8 +545,8 @@ public static function get_comment_types() {
 	 * @return boolean True if registered.
 	 */
 	public static function is_registered_comment_type( $slug ) {
-		$slug = strtolower( $slug );
-		$slug = sanitize_key( $slug );
+		$slug = \strtolower( $slug );
+		$slug = \sanitize_key( $slug );
 
 		return in_array( $slug, array_keys( self::get_comment_types() ), true );
```

Comment:
```suggestion
		$registered_comment_types = self::get_comment_types();

		return isset( $registered_comment_types[ $slug ] );
```
Would this work? I wonder if that's faster.